NapDatabaseExport/MainForm.cs

```diff
@@ -253,19 +259,24 @@ private void btnExport_Click (object sender, EventArgs e)
                             var row = new object [reader.FieldCount];
                             do
                             {
-                                for (int j = 0; j < reader.FieldCount; j++)
-                                    row[j] = reader.GetValue (j);
-
+                                for (int j = 0; j < reader.FieldCount; j++) {
+                                    try {
+                                        row[j] = reader.GetValue(j);
+                                    }
```

Comment:
Unnecessary new line

Reply:
Which line is unnecessary? I already put a space before function call parenthesis